README.rst

```diff
@@ -43,16 +43,10 @@ Getting started with development
     pip install -e .
     pip install -e <KOLIBRI-PERSEUS-PLUGIN-LOCAL-PATH>
     kolibri plugin kolibri_exercise_perseus_plugin enable
-    yarn install
 
-4. Make sure that this message **DID NOT** appear in your logging output of ``yarn install``::
+4. Within this repo, run the following command, again using your kolibri dev virtualenv::
```

Comment:
> Within this repo

The word 'this' is inherently ambiguous here. Recommend always specifying either "the kolibri repo" or "the perseus repo"